### ISSUE TITLE
"Allow Once" option for host access requests

### ISSUE THREAD

**Issue by Psychpsyo** [Neos-Metaverse/NeosPublic#3138]
### Is your tweak request related to a problem? Please describe.

Sometimes there's reasons for wanting to only allow host access for one specific request, rather than all requests to a given host. Or there's a host where I'd like to confirm every request individually instead of granting access once.

### Describe what would you like tweaked

Add an 'Allow Once' option to the host access popup.

### Describe alternatives you've considered

1. Removing the host from the list of allowed hosts every single time.
This has multiple issues:
-it's annoying.
-multiple requests in quick succession will be handled by only one confirmation dialogue.
-it's possible to forget this step.

2. Not adding this. (I don't see a reason for this.)

### Additional context

The 'Disallow Once' functionality already exists, through the use of the X button. Though this is neither explicitly mentioned nor is it all that obvious. A more explicitly labeled button could be added in addition to the one requested here.

**Comment by Frooxius** [comment]
Hmm... we could add something like this, but question is how exactly should this work? Should it allow all requests for day 10 seconds? 1 minute maybe? Even more/less?

To give a bit more context, what's your main motivation for allowing the access only once vs just leaving it?

**Comment by mralext20** [comment]
duplicate of #904 ?

**Comment by Psychpsyo** [comment]
> 
> 
> Hmm... we could add something like this, but question is how exactly should this work? Should it allow all requests for day 10 seconds? 1 minute maybe? Even more/less?
> 
> To give a bit more context, what's your main motivation for allowing the access only once vs just leaving it?

The point would be so that you can confirm every single request going to a specific host. (The popup would be valid only for confirming the single GET/POST request it was opened for)
For websockets it would probably make the most sense to allow that one connection to get established and any other connection needs to be reconfirmed. (or the same connection if it gets interrupted and needs to reconnect.)

But, as @mralext20 pointed out, this is a duplicate of #904 so I will close this issue and post the same elaboration over there.
